sdk/core/core-tracing/package.json

```diff
@@ -37,11 +37,8 @@
   },
   "files": [
     "dist/",
-    "dist-esm/lib/",
-    "src/",
-    "lib/",
-    "types/core-tracing.d.ts",
-    "ThirdPartyNotices.txt"
```

Comment:
Did you intentionally delete the thirdpartynotices?

Reply:
Yes Jeff mentioned that they don't use it anymore for this library

Reply:
Yeah, before OpenTelemetry had npm packages we had to inline their interfaces, but now we express this dependency externally.